refinery/config/settings/prod.py

```diff
@@ -6,6 +6,4 @@
 TEMPLATE_DEBUG = DEBUG
 
 # Required when DEBUG = False
-ALLOWED_HOSTS = get_setting("ALLOWED_HOSTS")
-
-EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
```

Comment:
@scottx611x : Looking at this again, I remember that we did need to add `console.EmailBackend` for testing, but in production I'd think we'd still want this?

Reply:
Yes, both are needed in production.

Reply:
@hackdna Doesn't the `EMAIL_BACKEND` in `aws.py` override this anyways?

We were wondering if this was even needed in `prod.py` if theres no accompanying SMTP server running by default.

Reply:
Yes. The purpose of a prod config is to run the site with real users which means you need a real SMTP backend to send email.

Reply:
Talking with Scott, realized that we were probably too fixated on running these tests in production mode. If we we ran it in development mode, the only real difference is that if something did go wrong we'd get the extra debug information, and that's a good thing.